src/foam/u2/view/MultiSelectView.js

```diff
@@ -1,207 +1,210 @@
 /**
  * @license
- * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * Copyright 2025 The FOAM Authors. All Rights Reserved.
  * http://www.apache.org/licenses/LICENSE-2.0
  */
 
 
 foam.CLASS({
-    package: 'foam.u2.view',
-    name: 'MultiSelectView',
-    extends: 'foam.u2.View',
-    flags: ['web'],
-    
-
-    documentation: `
-        The choices are in [value, label, isSelected ] quartets.
-
-        However the client can simply pass in [value, label] and it will adapt to a [value, label, isSelected, choiceMode ] format
-
-
-        Calling the following methods:
-
-        MultiChoiceView.data will be automatically set to a predicated dao based on the choices selected only if
-        the minSelected, maxSelected criteria is respected, it will be foam.dao.NullDAO othrewise
-    `,
-
-    css: `
-    ^input-container {
-        padding: 10px;
-        border: 1px solid $grey400;
-        cursor: pointer;
-        min-width: 400px;
-        display: flex;
-        align-items: center;
-        justify-content: space-between;
-        background-color: $white;
-        color: $surface-neutral-darker;
-        border-radius: 4px;
-    }
-    ^select-modal {
-        background: $white!important;
-        color: $black !important;
-        opacity: 1 !important;
-        border: 1px solid $grey400;
-        box-shadow: none !important;
-        border-radius: 4px;
-        margin-top: 5px;
-    }
-    ^choices-holder {
-        display: flex;
-        flex-direction: column;
-        gap: 2rem;
-        line-height: 1.2;
-        padding: 5px;
-    }
-    ^input-holder {
-        display: flex;
-        align-items: center;
-        gap: 1rem;
-        color: $black;
-        width: 100%;
-    }
-    ^checkbox-input {
-        -webkit-appearance: none;
-        appearance: none;
-        border-radius: 2px;
-        border: solid 1px $grey400;
-        height: 1.275em;
-        margin: 7px 0;
-        padding: 0px;
-        transition: background-color 140ms, border-color 140ms;
-        width: 1.275em;
-        flex-shrink: 0;
-    }
-    ^checkbox-input:checked {
-        background-color: $primary500;
-        border-color: $primary500;
-    }
-    ^checkbox-input:checked:disabled {
+
+  package: 'foam.u2.view',
+  name: 'MultiSelectView',
+  extends: 'foam.u2.View',
+  flags: ['web'],
+
+
+  documentation: `
+      The choices are in [value, label, isSelected ] quartets.
+
+      However the client can simply pass in [value, label] and it will adapt to a [value, label, isSelected, choiceMode ] format
+
+
+      Calling the following methods:
+
+      MultiChoiceView.data will be automatically set to a predicated dao based on the choices selected only if
+      the minSelected, maxSelected criteria is respected, it will be foam.dao.NullDAO othrewise
+  `,
+
+  css: `
+    .foam-u2-ActionView-options {
```

Comment:
Should scope with ^, ie. ^ .foam-u2-ActionView-options { 
in case any other view has an action called 'options'.